pipeline/coordinator.go

```diff
@@ -318,7 +318,7 @@ func checkMistCompatibleCodecs(strategy Strategy, iv video.InputVideo) Strategy
 // failures (today this means re-running the job in another pipeline). If it's
 // set to true, error callbacks from this job will not be sent.
 func (c *Coordinator) startOneUploadJob(p UploadJobPayload, handler Handler, foreground, hasFallback bool) <-chan bool {
-	if !foreground {
+	if !foreground || p.InFallbackMode {
```

Comment:
Isn't this going to break things by stopping callbacks to Studio in fallback mode though?

Reply:
Also maybe we should do something like `p.RequestID = fmt.Sprintf("fb_%s", p.RequestID)` to distinguish fallback requests from background and foreground ones

Reply:
Yes...yes it will. Fixing.